benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1239,6 +1329,19 @@ void GraphicsBenchmarkApp::DrawExtraInfo()
         ImGui::Text("%.2f GB", dataWriteInGb);
         ImGui::NextColumn();
 
+        const auto bandwidthLive = GetLiveStatistics(mMetricsData.metrics[MetricsData::kTypeBandwidth]);
+        ImGui::Text("Write Bandwidth");
+        ImGui::NextColumn();
+        ImGui::Text(
+            "%.2f GB/s min=%.2f max=%.2f\n%.2f%s%.2f GB/s",
```

Comment:
super super nit: `GiB`? (Not even sure, the wikipedia article is confusing as GB seems to be either 1000^3 or 1024^3 depending on context)

Reply:
In memory bandwidth context, `GB` means $2^{30}$. The places that use $10^9$ are network transfer and storage drives, because ... marketing...
Yes I guess I'll change this and the other existing parts to `GiB` to be extra clear